lib/IRGen/GenCall.cpp

```diff
@@ -4548,22 +4548,6 @@ void CallEmission::externalizeArguments(IRGenFunction &IGF, const Callee &callee
 
     bool isForwardableArgument = IGF.isForwardableArgument(i - firstParam);
 
-    // In Swift, values that are foreign references types will always be
-    // pointers. Additionally, we only import functions which use foreign
-    // reference types indirectly (as pointers), so we know in every case, if
-    // the argument type is a foreign reference type, the types will match up
-    // and we can simply use the input directly.
```

Comment:
This comment is no longer correct, since we now import `ImmortalRef *const &` as `ImmortalRef!`.